src/OpenLoco/GameState.h

```diff
@@ -144,6 +131,7 @@ namespace OpenLoco
         char userStrings[Limits::kMaxUserStrings][32];                           // 0x432A44 (0x0095885C)
         uint16_t routings[Limits::kMaxVehicles][Limits::kMaxRoutingsPerVehicle]; // 0x442A44 (0x0096885C)
         uint8_t orders[Limits::kMaxOrders];                                      // 0x461E44 (0x00987C5C)
+        LastWindowState lastWindowState;
```

Comment:
unfortunately reordering the GameState struct like this causes the game to lock on white screen on startup, I don't think this kind of reorganisation of GameState is possible

Reply:
yup. Its a pain but one day we will be able to. One day.